system-monitor@paradoxxx.zero.gmail.com/metadata.json

```diff
@@ -1,5 +1,5 @@
 {
-    "shell-version": ["3.26", "3.28", "3.30", "3.32", "3.34"],
+    "shell-version": ["3.26", "3.28", "3.30", "3.32", "3.34", "3.36"],
```

Comment:
Do you know whether we are still backwards compatible? i.e. Does this patch work as far back as 3.26? Otherwise we should remove incompatible versions.

Thanks!

Reply:
Hello,

After a few tests and after less than an hour using it, I can only say that this extension works on Gnome-shell 3.34 on Fedora 31 x86_64. I can't say anything about other Gnome-shell versions or other distributions.
Hope this will help to merge this PR soon, as Gnome-shell 3.36 is landing on distributions.

Cordially,

-- 
NVieville

Reply:
Thanks for the feedback. I guess I'll spin a Fedora 27 image when I have some time to test as far back as 3.26. If it works, then we can merge. If it doesn't, we'll have to figure out where the cutoff is.

Reply:
Btw, Fedora 27 has reached EOL long time in November 2018 and there are not package updates for it. I don't know which Gnome is in RHEL, though.

Reply:
Hello,

Not rock solid information as Redhat packages information is not so easy to get without having RHEL distribution at hand (sorry but I don't have one).

rhel 7: 3.22 [1]
rhel 8: 3.28 [2]
centos 7: 3.28 [3]
centos 8: 3.32 [3]

If necessary and if someone with RHELs at hand could confirm this, it would be very valuable.

-- 
NVieville
[1]: ftp://ftp.redhat.com/redhat/linux/enterprise/7Server/en/os/SRPMS/
[2]: ftp://ftp.redhat.com/redhat/rhel/rhel-8-beta/appstream/x86_64/Packages/
[3]: https://git.centos.org/rpms/gnome-shell/releases

Reply:
Tested in gnome-shell 3.26, 3.28, and 3.36 and it works (upon adding legacy support). I didn't test in between. I'm assuming it works in between.